### DISCUSSION TITLE
Windows build support

### DISCUSSION THREAD

**Discussion by jagadhees-ravi** [Eventual-Inc/Daft#331] [Q&A]
Hi, 
Can you please tell what are all the requirements when installing daft. I'm using python 3.9. 
The error I am getting while installing is 

"ERROR: Could not find a version that satisfies the requirement ray==1.13.0 (from getdaft) (from versions: none)
ERROR: No matching distribution found for ray==1.13.0".

**Comment by samster25** [comment]
Hi, Thanks for reaching out! The requirements for daft can be [found here](https://github.com/Eventual-Inc/Daft/blob/4b620cc3393abbf4d0a8f99407baa6b01049c3cf/pyproject.toml#L35). 

Did you mind sharing how you are installing daft?

**Comment by samster25** [comment]
Also would be great if you could share your operating system!

**Comment by jagadhees-ravi** [comment]
Hi,

I am using windows 11.
I am installing daft using "pip install getdaft" command in cmnd prmpt. Is there something i need to install or have on my pc before doing this?

**Comment by jaychia** [comment]
Hi @jagadhees-ravi - it seems that your setup is having trouble finding the version of Ray (which is a dependency of Daft) that satisfies its requirements.

I see that Ray has distributions for Windows but only for Python 3.7 onwards: https://pypi.org/project/ray/1.13.0/#files

What version of Python are you using?

**Comment by jagadhees-ravi** [comment]
Hi Jaychia, I am using Python 3.10 version. I have installed ray 1.13 also and it's still not working.
Below message is the new error i'm getting now.
"ERROR: Failed building wheel for getdaft
Failed to build getdaft
ERROR: Could not build wheels for getdaft, which is required to install pyproject.toml-based projects"

**Comment by jaychia** [comment]
> Hi Jaychia, I am using Python 3.10 version. I have installed ray 1.13 also and it's still not working. Below message is the new error i'm getting now. "ERROR: Failed building wheel for getdaft Failed to build getdaft ERROR: Could not build wheels for getdaft, which is required to install pyproject.toml-based projects"

Thanks @jagadhees-ravi, we have opened an issue #235 to build wheels for Windows so that you won't need to build from source. Your system is likely not equipped for building daft from source and that is why you are having this error. For now, if you have access to a Mac/Linux machine we would suggest that in the meantime.

Will update when ready!

**Comment by jagadhees-ravi** [comment]
Thanks @jaychia, Hope issue #235 will be resolved in no time.

**Comment by jaychia** [comment]
Migrated into a Github Discussion thread instead - please track #235 for progress/updates on Windows support!

**Comment by samster25** [comment]
@jagadhees-ravi, sorry for the wait but we now have daft wheels for windows!
